langchain/memory/__init__.py

```diff
@@ -1,3 +1,5 @@
+"""Memory manages and manipulates previous chat messages and incorporates
```

Comment:
nit chat messages suggests this only works for ChatModels, when it works generically for any chain. would just say "Memory maintains Chain state, allowing them to incorporate context from past runs."